### PR TITLE
refactor toast into separate components

### PR DESCRIPTION
**Pull request by radnan**

### CODE REVIEW

File: packages/react/src/toaster/Toaster.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    ref,
+  ) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    return (
+      <RadixToast.ToastProvider
+        duration={duration}
+        label={label}
+        swipeDirection={swipeDirection}
```
Comment (radnan) [inline packages/react/src/toaster/Toaster.tsx:39]: todo: need to set `swipeDirection` based on `position`

File: packages/react/src/toast/Toast.tsx [hunk 2]
```diff
@@ -35,78 +29,39 @@ const getIcon = (type: string) => {
...
-        </Box>
-      </RadixToast.ToastProvider>
+      <Paper
+        asChild
+        elevation="md"
+        {...styles.root({ type })}
+        {...sprinkleProps}
+      >
+        <RadixToast.Root
+          onOpenChange={onOpenChange}
```
Comment (radnan) [inline packages/react/src/toast/Toast.tsx:44]: todo: add animations

File: apps/storybook/src/feedback/Toast.stories.tsx [hunk 3]
```diff
@@ -1,118 +1,107 @@
...
-  render: ToastTemplate,
 };
 
-export const InteractiveTest: Story = {
-  args: {
-    children: "This is an interactive test toast message",
-    position: "bottom-right",
-    type: "info",
-  },
+export const Interactive: Story = {
```
Comment (toha993) [inline apps/storybook/src/feedback/Toast.stories.tsx:104]: Need to change the `text content` as its different from the default text.

Comment (radnan) [reply apps/storybook/src/feedback/Toast.stories.tsx:110]: addressed

**Pull request merged**